src/utils/redirect.ts

```diff
@@ -19,13 +17,6 @@ export const safeRedirect = ({
         return;
       }
 
-      // Navigate to callbackUrl without page refresh if we are in the same origin
```

Comment:
It would have been better if this was in a separate PR

Reply:
Will make another PR for this

Reply:
https://github.com/multiversx/mx-sdk-dapp/pull/944